test/unit/reports/create.test.js

```diff
@@ -0,0 +1,99 @@
+jest.mock('../../../app/reports/mi-report/get-events')
+const { getEvents: mockGetEvents } = require('../../../app/reports/mi-report/get-events')
+jest.mock('../../../app/reports/mi-report/group-events-by-correlation-id')
+const { groupEventsByCorrelationId: mockGroupEventsByCorrelationId } = require('../../../app/reports/mi-report/group-events-by-correlation-id')
+jest.mock('../../../app/reports/mi-report/order-grouped-events')
+const { orderGroupedEvents: mockOrderGroupedEvents } = require('../../../app/reports/mi-report/order-grouped-events')
+jest.mock('../../../app/reports/mi-report/get-report-lines')
+const { getReportLines: mockGetReportLines } = require('../../../app/reports/mi-report/get-report-lines')
+jest.mock('../../../app/reports/mi-report/convert-to-csv')
+const { convertToCSV: mockConvertToCSV } = require('../../../app/reports/mi-report/convert-to-csv')
+jest.mock('../../../app/storage')
+const { writeFile: mockWriteFile } = require('../../../app/storage')
+
+const { reportsConfig } = require('../../../app/config')
+
```

Comment:
Remove this space?

Reply:
Thought it was useful to keep them separate so it's easier to identify the target of the tests, but isolating the config from the mocks.

Added extra spaces to mocks though

Reply:
Yeah that makes sense - feel free to leave the config separate then :)